README.md

```diff
@@ -112,26 +112,12 @@ This fork will always stay open-source and maintained by the community.
 <p align="right"><a href="#top">⬆️ Back to top️</a></p>
 
 
-
-<!-- CONTACT -->
-## Contact
-
-Lucas Lee - playcover@lucas.icu
```

Comment:
Some sort of formal contact must be preserved. Discord is not a formal mean of communication.